src/nni_manager/training_service/local/localTrainingService.ts

```diff
@@ -380,9 +380,9 @@ class LocalTrainingService implements TrainingService {
         const envVariables: { key: string; value: string }[] = [
             { key: 'NNI_PLATFORM', value: 'local' },
             { key: 'NNI_EXP_ID', value: this.experimentId },
-            { key: 'NNI_SYS_DIR', value: trialJobDetail.workingDirectory },
+            { key: 'NNI_SYS_DIR', value: `'${trialJobDetail.workingDirectory}'` },
```

Comment:
I don't think the value of environment variables should to be quoted.
We should quote them when composing the command, not here.

Reply:
fixed.